sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -209,21 +209,21 @@
  * <h3>Sharding BigQuery output tables</h3>
  *
  * <p>A common use case is to dynamically generate BigQuery table names based on
- * the current window. To support this,
- * {@link BigQueryIO.Write#to(SerializableFunction)}
+ * the current window or element. To support this,
+ * {@link BigQueryIO.Write#to(BigQueryIO.TableWritePolicy)}
  * accepts a function mapping the current window to a tablespec. For example,
  * here's code that outputs daily tables to BigQuery:
  * <pre>{@code
  * PCollection<TableRow> quotes = ...
  * quotes.apply(Window.<TableRow>into(CalendarWindows.days(1)))
  *       .apply(BigQueryIO.Write
  *         .withSchema(schema)
- *         .to(new SerializableFunction<BoundedWindow, String>() {
- *           public String apply(BoundedWindow window) {
+ *         .to(new TableWritePolicy<String>() {
+ *           public String apply(Context window) {
```

Comment:
It's not a window. Please audit the rest of the file for other inconsistencies - I found too many to put individual comments on each.